File: CPP_Joc_Windows_Android/SH_Client_Win_Cpp_Cmake/App/src/base/view/dialog/util/DynamicTextButtonConfig.cpp
#include "DynamicTextButtonConfig.h"

using namespace base;

DynamicTextButtonConfig::DynamicTextButtonConfig(
	int id,
	std::string text, Color& tintColor, float alpha)
	: id(id),
	text(text), tintColor(tintColor), alpha(alpha)
{
	//void
}

DynamicTextButtonConfig::~DynamicTextButtonConfig() {
	//void
}

File: Java_Joc_Windows_Android/AndroidStudioProject/core/src/base/menu/util/DynamicTextButtonConfig.java
package base.menu.util;

/**
 * Created by Adrin on 8/20/2016.
 */
public class DynamicTextButtonConfig {

    public int id;

    public String text;
    public float rawTextWidthDPM;
    public int tintColor;
    public float alpha;

    public DynamicTextButtonConfig(int id, String text, float rawTextWidthDPM, int tintColor, float alpha) {
        super();

        this.id = id;

        this.text = text;
        this.rawTextWidthDPM = rawTextWidthDPM;
        this.tintColor = tintColor;
        this.alpha = alpha;
    }

}
